pages/1.10/monitoring/logging/logging-api-examples/index.md

```diff
@@ -63,9 +63,9 @@ curl -k -H "Authorization: token=${DCOS_AUTH_TOKEN}" "${DCOS_URL}/system/v1/agen
 # Container Logs
 
 **Important:**
-The following example requires journald task logging, which by default is [disabled](/1.10/monitoring/logging/logging-api/#compatibility).
+The following example requires journald task logging, which by default is [disabled](/1.10/monitoring/logging/logging-api/#compatibility). So container log files for older versions are only accessible via the [Mesos task sandbox files API](http://mesos.apache.org/documentation/latest/sandbox/).
```

Comment:
> The following example requires journald task logging, which by default is [disabled](/1.10/monitoring/logging/logging-api/#compatibility).

This sentence isn't true as you changed the script below from using the DC/OS API (`/system` endpoint) to Mesos API (`/mesos` endpoint). I think it worth keeping both scripts.